packages/react-app/src/views/QuadraticDiplomacyReward.jsx

```diff
@@ -1,135 +1,126 @@
-import {
-  Form,
-  Input,
-  Button,
-  Card,
-  Divider,
-  Space,
-  Typography,
-  notification
-} from 'antd'
-const { Text } = Typography;
+import React, { useState, useMemo, useCallback } from "react";
+import { Button, Card, Divider, Space, Typography, Badge, notification } from "antd";
+import { Address, EtherInput } from "../components";
+const { Text, Title } = Typography;
 const { ethers } = require("ethers");
 
-import React, {useEffect, useState} from "react"
-
 const REWARD_STATUS = {
-  PENDING: 'reward_status.pending',
-  COMPLETED: 'reward_status.completed',
-  FAILED: 'reward_status.failed',
-}
+  PENDING: "reward_status.pending",
+  COMPLETED: "reward_status.completed",
+  FAILED: "reward_status.failed",
+};
 
-export default function QuadraticDiplomacyReward({
-  userSigner,
-  votesEntries
-}) {
-  const [voteResults, setVoteResults] = useState({});
-  const [totalSqrtVotes, setTotalSqrtVotes] = useState(0);
+export default function QuadraticDiplomacyReward({ userSigner, votesEntries, price }) {
+  // local state
```

Comment:
About comments: I think comments should be used for clarification and adding some value, and could be omitted when they can be easily inferred from the actual code.